Package/Sublime Text Theme/Sublime Text Theme.sublime-syntax

```diff
@@ -0,0 +1,216 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: Sublime Text Theme (JSON)
+file_extensions:
+  - sublime-theme
+scope: source.json.sublime.theme
+
+contexts:
+  main:
+    - match: (?=\{)
+      push: main-mapping
+    - include: rules-sequence-pop
+    - include: expect-mapping-rest
+
+  main-mapping:
+    - match: \{
+      scope: punctuation.section.mapping.begin.json
+      push:
+        - meta_scope: meta.theme.collection.sublime-theme meta.mapping.json
+        - match: \}
+          scope: punctuation.section.mapping.end.json
+          set: only-comments
+        - match: (?=")
+          push: [in-mapping-expect-comma, main-key]
+        - include: expect-key-rest
+
+  main-key:
+    - clear_scopes: 1
+    - match: (")(rules)(")
+      scope: meta.mapping.key.json meta.main-key.sublime-theme string.quoted.double.json
+      captures:
+        1: punctuation.definition.string.begin.json
+        2: keyword.other.main.sublime-theme
+        3: punctuation.definition.string.end.json
+      set: [expect-rules-sequence-value, expect-colon]
+    - match: (")(variables)(")
+      scope: meta.mapping.key.json meta.main-key.sublime-theme string.quoted.double.json
+      captures:
+        1: punctuation.definition.string.begin.json
+        2: keyword.other.main.sublime-theme
+        3: punctuation.definition.string.end.json
+      set: [expect-variables-mapping-value, expect-colon]
+    - match: (")(name|author)(")
```

Comment:
Do themes really have an author field?
I mean, what does it do? If it has no special purpose, I'd rather have generic "doesn't do anything" highlighting than a `keyword` scope.

Same for the name key, actually.